src/OpenTelemetry.Instrumentation.AWS/TracerProviderBuilderExtensions.cs

```diff
@@ -15,11 +15,11 @@
 // </copyright>
 
 using System;
-using OpenTelemetry.Contrib.Instrumentation.AWS;
-using OpenTelemetry.Contrib.Instrumentation.AWS.Implementation;
+using OpenTelemetry.Instrumentation.AWS.Implementation;
 using OpenTelemetry.Internal;
+using OpenTelemetry.Trace;
 
-namespace OpenTelemetry.Trace;
+namespace OpenTelemetry.Instrumentation.AWS;
```

Comment:
Old namespace was here for a reason.
Easy way to call `AddAWSInstrumentation` without adding new using at the top of the file.

Reply:
I see. Reverted to the original namespace.